portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,62 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomizeImage() {
+  // The images directory contains 13 images, so generate a random index between
+  // 1 and 13.
+  const imageIndex = Math.floor(Math.random() * 24) + 1;
+  const imgUrl = 'images/jfew-' + imageIndex + '.jpg';
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const imgElement = document.createElement('img');
+  imgElement.src = imgUrl;
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const imageContainer = document.getElementById('random-image-container');
+  // Remove the previous image.
+  imageContainer.innerHTML = '';
+  imageContainer.appendChild(imgElement);
+}
+
+function openIntromodal() {
+  const button = document.getElementById("Intro");
```

Comment:
I don't think the `const button` and `const span` definitions are needed since you aren't using it. Looking up by element ID unnecessarily may make your page slower. This applies to all your methods.